README.md

```diff
@@ -1,5 +1,29 @@
-# alt-text-tracker
+# Alt text tracker
 
-The `alt.js` script runs every day to count the total number of images published to The Michigan Daily, and the number of images with alternative text. If a data file does not already exist, the script will retrieve counts since December 2022. Otherwise, the script will just retrieve data from within the last 24 hours, appending to the data file.
+Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/main/packages/create-svelte).
 
-The script is generally accurate, though it does introduce mistakes from time to time. You can correct these mistakes by deleting the `data.csv` file and rerunning `alt.js`.
+## Developing
+
+Install dependencies with `pnpm install`
+
+To set up the local database, run `pnpm run init`. It may take a minute to fetch all the data.
```

Comment:
Oof, it is not intended for the data to go past 2022-12-31, that is quite alarming. Maybe max(date) is behaving in a way I'm not expecting right now, I'm not sure. 
The prod D1 database also has around double the entries it used to have now. So I think something definitely went wrong when I used max(date). I'll look into that. 

It definitely could be useful to have migrations instead of this setup in the future. I think with only a year's of data it doesn't take as long, but as time passes it could definitely become less sustainable.

Reply:
RIP I am so sorry — don't listen to my recommendations ever again 💀💀

Reply:
all good - upon inspection it was completely my fault 💀